components/ipfs/ipfs_utils.cc

```diff
@@ -342,4 +367,24 @@ bool IsValidNodeFilename(const std::string& filename) {
   return RE2::FullMatch(filename, kExecutableRegEx);
 }
 
+GURL ContentHashToCIDv1URL(const std::string& contenthash) {
+  int64_t code = 0;
+  base::StringPiece input = contenthash;
```

Comment:
why not take StringPiece as a function argument

Reply:
it will be modified in DecodeVarInt